runner/runner.go

```diff
@@ -65,6 +65,30 @@ func (r *Runner) exec(raw, evt, path, dst string, user *users.User) error {
 		return err
 	}
 
+	envMapping := func(key string) string {
+		switch key {
+		case "FILE":
+			return path
+		case "SCOPE":
+			return user.Scope
+		case "TRIGGER":
+			return evt
+		case "USERNAME":
+			return user.Username
+		case "DESTINATION":
+			return dst
+		default:
+			return os.Getenv(key)
```

Comment:
I think using global env is not safe. Let's use an empty string as a default value